conf/cassandra.yaml

```diff
@@ -2368,3 +2368,104 @@ drop_compact_storage_enabled: false
 #   compatibility mode would no longer toggle behaviors as when it was running in the UPGRADING mode.
 #
 storage_compatibility_mode: NONE
+
+# Configuration for AutoRepair Scheduler
+#auto_repair:
+#  # Enable/Disable auto repair scheduler.
+#  # If it is set to false, then the scheduler thread will not be spun up.
+#  # If it is set to true, then the repair scheduler thread will be created. The thread will
+#  # look for secondary config available for each repair type (full, incremental, and preview_repaired),
+#  # and based on that, it will schedule repairs.
+#  enabled: false
+#  # Time interval between successive checks for repair scheduler to check if either the ongoing repair is completed or if
+#  # none is going, then check if it's time to schedule or wait.
+#  repair_check_interval: 5m
+#  # Maximum number of retries for a repair session.
+#  repair_max_retries: 3
+#  # Backoff time in seconds for retrying a repair session.
+#  repair_retry_backoff: 30s
+#  # Minimum duration for the execution of a single repair task (i.e.: RepairRunnable).
+#  # This helps prevent the auto-repair scheduler from overwhelming the node by scheduling a large number of
+#  # repair tasks in a short period of time.
+#  repair_task_min_duration: 5s
+#  # When any nodes leave the ring then the repair schedule needs to adjust the order, etc.
+#  # The AutoRepair scheduler keeps the deleted hosts information in its persisted metadata for the defined interval in this config.
+#  # This information is useful so the scheduler is absolutely sure that the node is indeed removed from the ring, and then it can adjust the repair schedule accordingly.
+#  # So, the duration in this config determinses for how long deleted host's information is kept in the scheduler's metadata.
+#  history_clear_delete_hosts_buffer_interval: 2h
+#  # Configuration specific to each repair type. Options: full, incremental, and preview_repaired.
+#  repair_type_overrides:
+#    full:
+#      # Enable/Disable full auto repair
+#      enabled: false
+#      # Minimum time in hours between repairing the same node again. This is useful for extremely tiny clusters, say 5 nodes, which finishes
```

Comment:
`Minimum time in hours` -> `Minimum duration`

Reply:
Done